consumer/consumer.go

```diff
@@ -338,6 +337,7 @@ func (c *Consumer) handleMetadataChange() {
 		}
 		group := arr[1]
 		c.delGroup(group)
+		log.ErrorLogger.WithField("group", group).Info("Group is stop, because metadata is changed")
```

Comment:
`Group is stop because metadata is changed` => `Group was stopped since the metadata has changed`